app/views/users/password_reset_confirmation.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container py-5">
+  <div class="row">
+    <div class="col-md-6 offset-md-3">
+      <div class="card card-outline-secondary">
+        <div class="card-header">
+          <h3 class="mb-0">Password Reset!</h3>
+        </div>
+        <div class="card-body">
+          <p class="lead"><strong>Your new password is set and you are now ready to log in.</p>
```

Comment:
Can we change this to something like "Your password has been updated."? This version feels too informal.